core/kazoo_stdlib/src/kz_maps.erl

```diff
@@ -73,15 +73,18 @@ merge_right(_K, {'both', _Left, Right}) -> {'ok', Right}.
 
 -spec get(term(), map()) -> term().
 get(Keys, Map) ->
-    get(Keys, Map, undefined).
+    get(Keys, Map, 'undefined').
 
 -spec get(term(), map(), term()) -> term().
 get([Key|Rest], Map, Default) ->
     case maps:get(Key, Map, {?MODULE, Default}) of
         {?MODULE, Default} ->
             Default;
-        NestedMap ->
-            get(Rest, NestedMap, Default)
+        #{} = NestedMap ->
+            get(Rest, NestedMap, Default);
+        Value when Rest =:= [] ->
+            Value;
+        _ -> Default
```

Comment:
why this change ? the current behaviour is the same as for kz_json, ie, it will crash if you try to get with list of keys where current key is not a map. if that's the case, then add a new method (find?) that will try/catch the get and return `Default`

Reply:
Honestly I don't remember why, I changed it months ago. Anyway it seems that I don't use it anywhere in this PR, so i'm reverting it.